src/components/DashboardImmersion/DashboardImmersionPresentation.js

```diff
@@ -17,6 +17,7 @@ function DashboardImmersionPresentation({
   setCurrentLabel,
   submitHandler,
 }) {
+  console.log({ data: queryResponse?.data })
```

Comment:
This console log should be removed

Reply:
If there are no other issues in this PR, I will remove this with my next PR.